Localize/lang/strings.json

```
@@ -2212,6 +2231,7 @@
   "epOMnV": "Shared",
   "esOQ5p": "No values matching your search",
   "euC/DV": "Array",
+  "evO2Ky": "No errors found :-)",
```

Is the punctuation at the end supposed to be an emoji? Do we have guidelines about appropriate uses?

```suggestion
  "evO2Ky": "No errors found.",
```

---

Accidentally left that in, I've removed it